gobcore/database/reader/postgresql.py

```diff
@@ -0,0 +1,22 @@
+from gobcore.exceptions import GOBException
+from psycopg2 import Error
+from psycopg2.extras import DictCursor
+
+
+def query_postgresql(connection, query: str):
+    """Query Postgres
+
+    :param connection:
+    :param query:
+    :return:
+    """
+    try:
+        cursor = connection.cursor(cursor_factory=DictCursor)
```

Comment:
This can be written using a resource manager and the close cursor can be removed.
with connection.cursor(cursor_factory=DictCursor) as cursor:
    ....